### ISSUE TITLE
Throws error with given version number

### ISSUE THREAD

**Issue by addisaden** [ajyoon/bml#8]
Hello,

thank you for your great work. I really like the creativity in this language.

I found an error on executing a BML-File.

My file is this:

```
eval {
    settings = {
        renderMarkdown: false,
        version: "0.0.15-dev",
    };
}

Hello {(BML-Lovers), (creative authors), (poetry slammers)}
```
Output is:
```
no bml version specified in settings, unexpected behavior may occur.
Hello BML-Lovers
```

When i change the version to `0.0.14` (served by `npm install -g bml`) and run it on the current master it outputs this:
```
BML VERSION MISMATCH. bml source file specifies version 0.0.14 but running version is 0.0.15-dev. unexpected behavior may occur.
no bml version specified in settings, unexpected behavior may occur.
Hello creative authors
```

**Comment by ajyoon** [comment]
Hey thanks so much for the report, and apologies for the delayed reply. This bug was happening because BML recursively renders choice outputs (`"BML-Lovers"` in your example), which don't contain the version numbers it wanted. But actually, I think in general it's more noise than it's worth to warn on absent versions, so as of f6e8c8872f3a2eb7150e2f92d466c53c96d160d4 (to be released in `0.0.16`) BML will no longer issue that warning. I'll close this issue once `0.0.16` is cut.

**Comment by ajyoon** [comment]
0.0.16 is out now, so this error should be fixed. thanks!
